magenta-lib/src/main/scala/magenta/input/resolver/DeploymentTypeResolver.scala

```diff
@@ -23,9 +23,18 @@ object DeploymentTypeResolver {
     invalidActions.map(invalids => Invalid(
       ConfigErrors(deployment.name, s"Invalid action ${invalids.toList.mkString(", ")} for type ${deployment.`type`}"))
     ).getOrElse {
-      import henkan.convert.Syntax._
       Validated.fromOption(actions.map(as =>
-        deployment.to[Deployment].set(actions = as)
+        Deployment(
+          name = deployment.name,
+          `type` = deployment.`type`,
+          stacks = deployment.stacks,
+          regions = deployment.regions,
+          actions = as,
+          app = deployment.app,
+          contentDirectory = deployment.contentDirectory,
+          dependencies = deployment.dependencies,
+          parameters = deployment.parameters,
+        )
```

Comment:
Is this the result of the removal of `henkan.convert.Syntax._`?

Reply:
Yep. We only had a couple of use cases, so it seemed better to just write it out. In general, because of Scala's tendency for difficult upgrades, I recommend only adding third-party dependencies when there is a large and clear benefit.